New version: LibAwsIO v1.2.12

JuliaRegistrator opened the pull request:
- Registering package: LibAwsIO
- Repository: https://github.com/JuliaServices/LibAwsIO.jl
- Created by: @quinnj
- Version: v1.2.12
- Commit: a33561b337c4c2612c9d5430d5dfa59e5fc46457
- Reviewed by: @quinnj
- Reference: https://github.com/JuliaServices/LibAwsIO.jl/commit/a33561b337c4c2612c9d5430d5dfa59e5fc46457#commitcomment-172431118
- Description: Julia wrapper for the aws-c-io library.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd639700582975867f79fef7e04f261b53b05b22d59a1d0459b49e3cda2dd7bf1c18542433e14b83bff0a2c8fbe769400504f6c3566a46a200e26e2c1d6bf44df2d5fe360a08d4d666c933c51a0896ff5549292b7bbe95ec1f848a53e28e5f5fec8f11af1df5ff4376ed23e7743f6c751b52adfb2c3e8397058cb6079493889353d1619b1c2c8c0497f705e29938c2de5da2a50da3c310f8cfbf220e4787c39051b411ce0453108764d027a0b0d8bfd33f0 -->

quinnj commented:
[noblock] can someone help resolve the conflict here? I made a new release (1.3) and now this backport release is conflicting (they were made around the same time, so i think they were both based off teh same base branch of General registry)

JuliaRegistrator committed:
New version: LibAwsIO v1.2.12

UUID: a5388770-19df-4151-b103-3d71de896ddf
Repo: https://github.com/JuliaServices/LibAwsIO.jl.git
Tree: 7a2da0e4f4c91b6387e06be34930a3f2a4e5d68c

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
